Files/ViewModels/Widgets/IWidgetItemModel.cs

```diff
@@ -5,5 +5,7 @@ namespace Files.ViewModels.Widgets
     public interface IWidgetItemModel : IDisposable
     {
         string WidgetName { get; }
+
+        bool IsWidgetSettingEnabled { get; }
```

Comment:
What's this property used for?

Reply:
It determines whether a setting for the widget is enabled. The name is pretty straight forward..